### ISSUE TITLE
Start doesn't do anything

### ISSUE THREAD

**Issue by FelixLttks** [vitaminwater/ChatGPT-SelfChat#1]
after installing the extension via chrome webstore, setting all 3 tabs (subject, talker 1 & 2) up and pressing start, nothing happens

Edit: ChatGPT is down. Maybe that's the cause of the error

**Comment by vitaminwater** [comment]
Not sure which was the issue, the missing part in the manifest, or if I uploaded a wrong archive. Seemed that it worked on my side at one point from chrome. Anyway, it should work now thanks.

**Comment by mat926** [comment]
I'm having the same issue

**Comment by vitaminwater** [comment]
Yes actually the problem is still there, but with a delay, that's why I didn't see it. In the meantime the firefox version works perfectly.

**Comment by billsargent** [comment]
> Yes actually the problem is still there, but with a delay, that's why I didn't see it. In the meantime the firefox version works perfectly.

Not anymore. Two instances of firefox open and then neither talk.

**Comment by ghost** [comment]
> > Yes actually the problem is still there, but with a delay, that's why I didn't see it. In the meantime the firefox version works perfectly.
> 
> Not anymore. Two instances of firefox open and then neither talk.

same
but for me the resolution of the windows is bigger than my screen
